styling/Phoenix.html

```diff
@@ -35,7 +35,23 @@
+--></div></div></div></div></header>
+
+
+<div class="container body-container"><div><div id="subheader-columns"><div class="row">
+<div class="col-xs-4 subheader"><h4>How is Phoenix different?</h4>
+<p>Phoenix is a framework for building HTML5 apps, API backends and distributed systems. Written in Elixir, you get a beautiful syntax, a productive tooling and fast runtime.
+<a href="http://www.phoenixframework.org/v0.10.0/docs">Get started with Phoenix</a></p></div>
+<div class="col-xs-4 subheader"><h4>Building the new web</h4>
+<p>Channels provide real-time streaming directly within Phoenix, allowing us to build rich, interactive applications, be it in the browser, a native mobile app or in embedded devices.
```

Comment:
`<strong>real-time streaming directly within Phoenix</strong>`

Also what do we mean "directly within Phoenix." Maybe that needs clarification?